modular_skyrat/modules/primitive_cooking_additions/code/big_mortar.dm

```diff
@@ -59,6 +59,29 @@
 	return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
 
 /obj/structure/large_mortar/attackby(obj/item/attacking_item, mob/living/carbon/human/user)
+	if(istype(attacking_item, /obj/item/storage/bag))
+		if(length(contents) >= maximum_contained_items)
+			balloon_alert(user, "already full")
+			return TRUE
+		
+		if(!attacking_item.contents.len)
+			balloon_alert(user, "bag empty")
+			return TRUE
+			
+		for(var/obj/item/target_item in attacking_item.contents)
+			if(length(contents) >= maximum_contained_items)
+				break
+				
+			if(target_item.juice_results || target_item.grind_results)
+				target_item.forceMove(src)
+
+		if (length(contents) >= maximum_contained_items)
+			balloon_alert(user, "mortar filled")
+		else
+			balloon_alert(user, "bag emptied")
```

Comment:
That's exactly the behavior shown when dumping a plant bag into a biogenerator, yes. It's a little weird but I wanted to keep it consistent between functionally similar structures. I also avoided using [attacking_item] and [src] for reason of keeping balloon text short and sweet because 'large mortar filled' and 'portable seed extractor emptied' were just a little too wordy for my tastes, but that's fine.